libtlafmt/src/ast_format/node.rs

```diff
@@ -682,6 +682,55 @@ X == A \/ B
 ---- MODULE B ----
 X == "bananas"
 ====
+"#
+        );
+    }
+
+    #[test]
+    fn test_spec_always_true_ident() {
+        assert_rewrite!(
+            r#"
+---- MODULE B ----
+LEMMA TypeCorrect == Spec => []TypeInv
+THEOREM DeadlockFreedom == Spec => [] Invariant
+====
+"#
+        );
+    }
```

Comment:
This covers some painful spacing emitted `[] Here` with an isolated regression test.